src/sys/unix/uds/mod.rs

```diff
@@ -103,6 +104,7 @@ cfg_os_poll! {
             target_os = "watchos",
             target_os = "espidf",
             target_os = "vita",
+            target_os = "nto",
         ))]
         {
             syscall!(fcntl(fds[0], libc::F_SETFL, libc::O_NONBLOCK))?;
```

Comment:
SOCK_CLOEXEC is already supported as part of the socketpair() call.  Can skip the FD_CLOEXEC calls below.